src/arithmetic/rewrite_simplify.cc

```diff
@@ -1674,6 +1674,14 @@ Mutate_(const Call* op, const Expr& self) {
   if (op == nullptr) return ret;
   if (op->is_intrinsic(Call::likely) && is_const(op->args[0])) {
     return op->args[0];
+  } else if (op->is_intrinsic(Call::shift_right)) {
+    if (op->args[0].as<IntImm>() && op->args[1].as<IntImm>()) {
+      return op->args[0] >> op->args[1];
```

Comment:
Do we have `shift_right` defined on the `IntImm` ? Should it be `op->args[0]->value >> op->args[1]->value` ? 
Same for the `and`.

Reply:
the construction itself will constant fold, i will add a followup comment.

Reply:
Oh, I see. Okay, thanks.